lib/active_remote/persistence.rb

```diff
@@ -245,9 +245,9 @@ def create
     # are created, existing records are updated. If the record is marked as
     # readonly, an ActiveRemote::ReadOnlyRemoteRecord is raised.
     #
-    def create_or_update
+    def create_or_update(*args)
       raise ReadOnlyRemoteRecord if readonly?
-      new_record? ? create : update
+      new_record? ? create : update(*args)
```

Comment:
Actually, after looking at this again, I'm not sure that either of these need to take args. Is there some AR functionality that this is replicating that I'm not aware of?